rcl_logging_spdlog/README.md

```diff
@@ -2,15 +2,15 @@
 
 Package supporting an implementation of logging functionality using `spdlog`.
 
-The `rcl_logging` interface allows [rcl_logging_spdlog/logging_interface.h](include/rcl_logging_spdlog/logging_interface.h):
+[rcl_logging_spdlog](include/rcl_logging_spdlog/logging_interface.h):
```

Comment:
@ahcorde if we are going to keep a (probably duplicate) logging API enumeration, maybe something like: 
```suggestion
[rcl_logging_spdlog](include/rcl_logging_spdlog/logging_interface.h) logging interface implementation can:
```
would read better. Feel free to rephrase, I just made this one up off the top of my head.